prover/drivers/kore-driver.md

```diff
@@ -47,7 +47,9 @@ Add various standard Kore declarations to the configuration directly:
          ...
        </declarations>
 
-  rule <k> (axiom _ #as DECL:Declaration) => .K ... </k>
+  rule axiom (P:Pattern) => axiom (!N:AxiomName) : P
+
+  rule <k> (axiom _: _ #as DECL:Declaration) => .K ... </k>
```

Comment:
What does this rule do?

Reply:
The first rule creates a fresh name of shape `axN` for an unnamed axiom declaration - see [here](https://github.com/kframework/matching-logic-prover/pull/32/files#diff-83b0f9eff1bb6916149ec00613a9b05aR691-R694). The second rule moves named axiom into declarations.